zammad/templates/cronjob-reindex.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "zammad.fullname" . }}-cronjob-reindex
+  labels:
+    {{- include "zammad.labels" . | nindent 4 }}
+    app.kubernetes.io/component: zammad-cronjob-reindex
+  annotations:
+    description: |-
+      This cronjob never runs. It can be used to create maintenance jobs with
+      kubectl create job my-reindex-job --from=cronjob/{{ include "zammad.fullname" . }}-cronjob-reindex
+    {{- include "zammad.annotations" . | nindent 4 }}
+  {{- with .Values.zammadConfig.cronJob.reindex.annotations }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  # Never run this cronjob.
```

Comment:
Couldn't `suspend` and `schedule` be made configurable? We are currently considering whether we want to do a reindex on Sunday night after all